tests/test_filtsmooth/filtsmooth_testcases.py

```diff
@@ -6,12 +6,11 @@
 import probnum.diffeq as pnd  # ODE problem as test function
 import probnum.filtsmooth as pnfs
 import probnum.statespace as pnss
+from probnum.problems import RegressionProblem
 from probnum.randvars import Constant, Normal
```

Comment:
could you maybe change the imports here to `from probnum import diffeq, filtsmooth, statespace, problems, randvars` and update accordingly? Or would this be out of scope for this PR?

Reply:
Will do